src/uct/cuda/gdr_copy/gdr_copy_md.h

```diff
@@ -29,11 +29,13 @@ typedef struct {
  * gdr copy domain configuration.
  */
 typedef struct uct_gdr_copy_md_config {
-    uct_md_config_t         super;
-    int                     shared;       /**< Shared MD instance */
-    int                     enable_rcache;/**< Enable registration cache */
-    ucs_linear_func_t       uc_reg_cost;  /**< Memory registration cost estimation
-                                             without using the cache */
+    uct_md_config_t   super;
+    int               shared;          /**< Shared MD instance */
+    int               enable_rcache;   /**< Enable registration cache */
+    ucs_linear_func_t uc_reg_cost;     /**< Memory registration cost estimation
+                                            without using the cache */
+    double            rcache_overhead; /**< Registration cache lookup overhead
```

Comment:
maybe we could add the whole ucs_rcache_config_t structure and use ucs_rcache_set_params, and config as used here: https://github.com/openucx/ucx/blob/55fb3751df90a923c6c7d3aebe2ad9e2495b306f/src/ucp/core/ucp_context.c#L646

Reply:
Yes, we could. Thought it would introduce too many extra parameters.